learn/tanssi/technical.md

```diff
@@ -3,51 +3,83 @@ title: Technical Features
\ No newline at end of file
+### Tanssi Network {: #tanssi-newtwork } 
+
+As previously discussed, Tanssi Protocol assigns collators to Tanssi Network itself and the ContainerChains, and the result of this assignment is stored within the chain state.
+
+Another important piece of information that Tanssi stores is the latest header for every ContainerChain. This data is read from the relay chain and, being stored in every Tanssi block, it allows the protocol to keep track of the state in every chain and also to identify and reward accordingly the collator that produced their last block.
+
+### ContainerChain {: #containerchain } 
+
+As a collator node assigned to a ContainerChain deployed in Tanssi, and having a built-in Tanssi node functionality, it is technically feasible to read the state from the relay chain and the state from the Tanssi Network.
+
+Leveraging this ability to access the states, the current collator with the authority to produce a block will read the latest block produced in the relay chain which contains the state root of the latest block produced in Tanssi. With this state root, it will proceed to read the state in Tanssi, and include in the block of the ContainerChain the latest state root of Tanssi Network, the current set of collators assigned to the ContainerChain, and its public signature, allowing Tanssi to know who produced the block and reward the collator.
+
+Once the block is completed with the ContainerChain transactions, it will be proposed, validated, and finalized by the relay chain, which will include the block's header in the relay chain block.
+
+## Implementing Tanssi Protocol in your ContainerChain 
+
+To make your Appchain Tanssi compliant and ready to become a deployed ContainerChain, adding references to the following two pallets is required:
+
+-**Authorities Noting pallet** - this pallet has the objective of implementing the necessary logic to read and include in the ContainerChain block the set of collators assigned to provide block production services in the current session
+
+-**Author Inherent pallet** - this pallet is necessary to allow the collator to include in the block its identity and be recognized and awarded as the block producer
+
+It is important to note that both pallets include the mentioned data in the block using Inherents, which are a special form of transaction that only the block producer can include.
+
+## Deploy a ContainerChain {: #deploy-a-containerchain } 
+
+After building on top of one of the provided [Appchain Templates](/learn/tanssi/templates) and finishing the development process, developers are ready to deploy their ContainerChain in Tanssi.
+
+This a fairly straightforward step, where the teams need only to generate and upload the [chain specification](https://docs.substrate.io/build/chain-spec/){target=blank} to the Tanssi orchestrator.
```

Comment:
Is the only purpose of the Tanssi orchestrator to assign a set of collators? Since this is the first time mentioning it, it might be nice to add a sentence or two to briefly describe what the orchestrator is

Reply:
I started using "tanssi orchestrator" as a synonym for the tanssi network. I changed it back :)